src/test/ui/argument-suggestions/extra_arguments.stderr

```diff
@@ -17,10 +17,10 @@ error[E0059]: arguments to this function are incorrect
   --> $DIR/extra_arguments.rs:9:14
    |
 LL |   one_arg(1, 1);
-   |              ^
-   |              |
-   |              no parameter of type {integer} is needed one_arg
-   |              help: removing this argument may help
+   |            --^
```

Comment:
This span looks wrong.

Reply:
It's technically "correct", because we're removing the , from the previous line; I'm still fiddling with all of this, and it was already pretty late last night when I got this working.  Today I'll probably change it to prefer the *following* comma when applicable, rather than the previous one.